docs/js/matomo.js

```diff
@@ -0,0 +1,21 @@
+// don't trigger analytics for local development environments
+if (document.domain !== 'localhost') {
```

Comment:
Shouldn't we prevent the trigger for netlify preview like https://deploy-preview-3789--mocha.netlify.com/ ?

Only trigger analytics in production site like:

```js
if  (document.domain.indexOf('mochajs.org') !== -1)
```

Reply:
good idea